content/gettingstarted/index.adoc

```diff
@@ -22,123 +22,125 @@ Kiali will support the latest Istio version (1.1 at the moment) and the developm
 
-Now install the latest release of Kiali.
+icon:lightbulb[size=1x]{nbsp} The Kiali Operator install script can be configured by setting environment variables when running it. For example, if you do not want it to install a Kiali CR (and thus avoid having the Kiali Operator install Kiali itself), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script. Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available to you.
 
-icon:lightbulb[size=1x]{nbsp} The install script can be configured for a customized install by setting environment variables when running the script. For example, to install a different version of Kiali, set IMAGE_VERSION to the version you want. Also, if your Jaeger or Grafana are accessible at different URLs than the default, then set JAEGER_URL and GRAFANA_URL appropriately. The defaults are: `JAEGER_URL="http://jaeger-query-istio-system.127.0.0.1.nip.io"` and `GRAFANA_URL="http://grafana-istio-system.127.0.0.1.nip.io"`
+icon:bullhorn[size=1x]{nbsp} The install script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
 
-icon:bullhorn[size=1x]{nbsp} This script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
+To install the Kiali Operator (and, by default, the Kiali CR), run the install script via this command:
 
 [source,bash]
 ----
-bash <(curl -L http://git.io/getLatestKialiKubernetes)
+bash <(curl -L https://git.io/getLatestKialiOperator)
 ----
 
-By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKialiKubernetes[the comments at the top of the install script].
+You will be prompted for the authentication strategy you want to use. Choose "login" if you want to define a username and password that users will be required to access Kiali. Choose "anonymous" to allow full access to Kiali without requiring any credentials. Choose "openshift" to use the OpenShift OAuth login which controls access based on the individual's RBAC roles in OpenShift (this is only a feature available when deployed on OpenShift).
```

Comment:
what happens on k8s if someone chooses openshift? or they won't have this option/it will output an error?

Reply:
I added a blurb here to explain what happens.